newIDE/app/public/external/piskel/piskel-main.js

```diff
@@ -364,52 +343,50 @@ ipcRenderer.on('piskel-load-animation', (event, receivedOptions) => {
 
   // Load a custom save file(s) header
   const pathEditorHeaderDiv = document.getElementById('path-editor-header');
-  const initialResourcePath =
-    receivedOptions.resources[0] === undefined
-      ? ''
-      : receivedOptions.resources[0].resourcePath;
-
   const savePathEditor = createPathEditorHeader({
     parentElement: pathEditorHeaderDiv,
     editorContentDocument: document,
     onSaveToGd: saveToGD,
     onCancelChanges: closeWindow,
-    projectPath: receivedOptions.projectPath,
-    initialResourcePath,
-    name: receivedOptions.name,
-    extension: piskelOptions.singleFrame ? '.png' : undefined,
+    name: externalEditorInput.name,
   });
 
-  if (!pskl) {
-    return;
-  }
   // Set piskel to tiled mode when editing a singleFrame object
   pskl.UserSettings.set(
     pskl.UserSettings.SEAMLESS_MODE,
-    piskelOptions.singleFrame
+    externalEditorInput.singleFrame
   );
 
   electronWindow.setTitle(
-    'GDevelop Pixel Editor (Piskel) - ' +
-      path.normalize(
-        !piskelOptions.singleFrame
-          ? receivedOptions.projectPath + '/' + receivedOptions.name
-          : initialResourcePath
-      )
+    'GDevelop Pixel Editor (Piskel) - ' + externalEditorInput.name
   );
 
   // If there were no resources sent by GD, create an empty piskel document
-  if (receivedOptions.resources.length === 0) {
-    piskelCreateAnimation();
-  } else if (piskelOptions.externalEditorData.pskl) {
+  if (externalEditorInput.resources.length === 0) {
+    createEmptyAnimation();
+  } else if (externalEditorInput.externalEditorData) {
+    console.log(externalEditorInput.externalEditorData)
     // If there is metadata from GD, use it to load the pskl document with frames with layers
     // Note that metadata will be saved only if the user has more than one layers
-    loadPiskelDataFromGd();
+    loadPiskelDataFromGd(externalEditorInput);
+
+    // Disable changing path and naming convention by user - on animations imported from gdevelop
+    savePathEditor.disableNameInput();
   } else {
     // If there are resources, but no metadata, load the images that were received from GD
-    loadImagesIntoPiskel();
+    loadImagesIntoPiskel(externalEditorInput);
+
     // Disable changing path and naming convention by user - on animations imported from gdevelop
-    savePathEditor.disableSavePathControls();
+    savePathEditor.disableNameInput();
   }
-  updateFrameElements();
+
+  setTimeout(() => {
```

Comment:
this used to have a function name "updateFrameElements".
Maybe add a comment to explain what this timeout does?

Reply:
Done 👍